tests/data/site_root/input/ImplementationGuide-KidsFirst.json

```diff
@@ -0,0 +1,72 @@
+{
+    "contact": [
+        {
+            "name": "Kids First DRC",
+            "telecom": [
+                {
+                    "system": "email",
+                    "value": "mailto:support@kidsfirstdrc.org"
+                }
+            ]
+        }
+    ],
+    "definition": {
+        "page": {
+            "generation": "html",
+            "nameUrl": "toc.html",
+            "page": [
+                {
+                    "generation": "markdown",
+                    "nameUrl": "index.html",
+                    "title": "Kids First Implementation Guide"
+                }
+            ],
+            "title": "Table of Contents"
+        },
+        "parameter": [
+            {
+                "code": "copyrightyear",
+                "value": "2020+"
```

Comment:
`"value"` mismatches with `tests/data/site_root/ig.ini` (2020+ vs. 2019+).

Reply:
Oh maybe its bc I copy pasted the IG JSON from somewhere else. Will fix